src/Datadog.Trace.ClrProfiler.Native/integration.h

```diff
@@ -165,6 +165,8 @@ struct MethodSignature {
     if (data.size() > 1) {
       return data[2] == ELEMENT_TYPE_OBJECT;
     }
+
+    return false;
```

Comment:
haha, C++ just lets you not return a value and just issues a warning? Good fix! 👍